components/scream/src/share/property_checks/field_lower_bound_check.hpp

```diff
@@ -0,0 +1,43 @@
+#ifndef SCREAM_FIELD_LOWER_BOUND_CHECK_HPP
+#define SCREAM_FIELD_LOWER_BOUND_CHECK_HPP
+
+#include "share/property_checks/field_within_interval_check.hpp"
+
+#include <limits>
+
+namespace scream
+{
+
+// Convenience implementation of check for interval [U,-\infty). The class
+// inherits from FieldWithinIntervalCheck, and sets lower bound to -infinity
+class FieldLowerBoundCheck: public FieldWithinIntervalCheck {
+public:
+  // Constructor with lower bound. By default, this property check
+  // can repair fields that fail the check by overwriting nonpositive values
+  // with the given lower bound. If can_repair is false, the check cannot
+  // apply repairs to the field.
```

Comment:
What about `allow_failures`?

Reply:
`allow_failures` only applies to the check phase. If f<0, check would return Warn, but repair would still attempt to repair the field.

In a way, allow_failure is more for the benefit of AtmosphereProcess. Maybe I should remove this enum from the PropertyCheck class, and add `allow_failure` to the methods `add_x_property_chec` of AtmosphereProcess?

Reply:
That sounds good to me.